doc/DOCUMENTATION.org

```diff
@@ -642,6 +651,24 @@ in =evil-hybrid-state-map= keymap instead of =evil-insert-state-map=.
 Hybrid mode can be enabled by setting =dotspacemacs-editing-style= to =hybrid=.
 To switch between evil and hybrid mode use ~SPC t E y~ and ~M-m t E y~.
 
+** Evilified modes
+Some buffers (such as Magit, for using git from within Emacs), are not for
+editing text, and provide their own keybindings for certain operations. These
+often conflict with Vim bindings. To make such buffers behave Vim-like in a
+consistent manner, they use a special state called /evilified/ state. In
+evilified state, a handful of keys work as in Evil, namely =/=, =:=, =h=, =j=,
+=k=, =l=, =n=, =N=, =v=, =V=, =gg=, =G=, =C-f=, =C-b=, =C-d=, =C-u= and =C-z=.
+All other keys work as intended by the underlying mode.
+
+Shadowed keys are moved according to the pattern: =a= → =A= → =C-a= → =C-A=
+
+For example, if the mode binds a function to =n=, that is found under =C-n= in
```

Comment:
Oh actually it is `C-n` but you should explain again why just to be explicit and make sure that the reader understood.